plugboard/cli/process/__init__.py

```diff
@@ -68,3 +71,27 @@ def run(
         progress.update(task, description=f"Running process...")
         asyncio.run(_run_process(process))
         progress.update(task, description=f"[green]Process complete[/green]")
+
+
+@app.command()
+def diagram(
+    config: Annotated[
+        Path,
+        typer.Argument(
+            exists=True,
+            file_okay=True,
+            dir_okay=False,
+            writable=False,
+            readable=True,
+            resolve_path=True,
+            help="Path to the YAML configuration file.",
+        ),
+    ],
+) -> None:
+    """Create a diagram of a Plugboard process."""
+    config_spec = _read_yaml(config)
+    with add_sys_path(config.parent):
+        process = _build_process(config_spec)
+    diagram = MermaidDiagram.from_process(process)
+    md = Markdown(f"```\n{diagram.diagram}\n```\n[Editable diagram]({diagram.url}) (external link)")
```

Comment:
Would it make sense to use the helper function here to simplify the code a bit?
```suggestion
    md = markdown_diagram(process)
```

Reply:
This is actually to do a bit more work than `markdown_diagram` - on the CLI it will also output a mermaid.live link as well so that you can open a version of the diagram in the browser.